src/core/classes/Sendmail.php

```diff
@@ -9,7 +9,7 @@ function __construct($args)
     $args['subject'] = $args['subject']?? "Message from ".Gila::config('base');
     $args['headers'] = $args['headers']?? "From: GilaCMS <noreply@{$_SERVER['HTTP_HOST']}>";
     $args['message'] = $args['message']?? "";
-    if($args['message']=="") foreach(@$args['post'] as $key) {
+    if($args['message']==="") foreach(@$args['post'] as $key) {
```

Comment:
you could also remove the error suppressing by doing `foreach($args['post'] ?? [] as $key)`